.github/workflows/ci.yml

```diff
@@ -146,7 +211,6 @@ jobs:
       - name: Setup Environment
         run: |
           yarn install
-
```

Comment:
please keep the new line, linting is expected everywhere, even the ci files :)

Reply:
Done